boot/makebootable.go

```diff
@@ -224,15 +224,12 @@ func makeBootable20(model *asserts.Model, rootdir string, bootWith *BootableSet)
 }
 
 func makeBootable20RunMode(model *asserts.Model, rootdir string, bootWith *BootableSet) error {
-	// XXX: move to dirs ?
-	runMnt := filepath.Join(rootdir, "/run/mnt/")
-
 	// TODO:UC20:
 	// - create grub.cfg instead of using the gadget one
 
 	// copy kernel/base into the ubuntu-data partition
-	ubuntuDataMnt := filepath.Join(runMnt, "ubuntu-data")
-	snapBlobDir := dirs.SnapBlobDirUnder(filepath.Join(ubuntuDataMnt, "system-data"))
+	systemData := filepath.Join(dirs.EarlyBootUbuntuData, "system-data")
```

Comment:
I forgot we do this, we need to think a bit, it's not bad in itself,  but I'm not sure having them defined in dirs is good

Reply:
as mentioned above, we resolved this in the previous PR so I think this will be cleaner and make more sense because makebootable will use boot pkg exported vars instead